x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/threshold_rule_ess_basic.cy.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ALERT_SUPPRESSION_DURATION_INPUT,
+  THRESHOLD_ENABLE_SUPPRESSION_CHECKBOX,
+} from '../../../screens/create_new_rule';
+
+import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
+import { selectThresholdRuleType } from '../../../tasks/create_new_rule';
+import { login } from '../../../tasks/login';
+import { visit } from '../../../tasks/navigation';
+import { CREATE_RULE_URL } from '../../../urls/navigation';
+import { TOOLTIP } from '../../../screens/common';
+
+describe('Threshold rules, ESS basic license', { tags: ['@ess'] }, () => {
+  beforeEach(() => {
+    deleteAlertsAndRules();
+    login();
+    visit(CREATE_RULE_URL);
+
+    // downgrade license to basic
+    cy.request({
```

Comment:
This should not be hardcoded here, it should be abstracted to a task so everybody can reuse it :)

Reply:
moved to the utilities API calls